bugsnag-android-core/src/main/java/com/bugsnag/android/Client.java

```diff
@@ -340,6 +340,7 @@ public Unit invoke(String oldOrientation, String newOrientation) {
                 }, new Function1<Boolean, Unit>() {
                     @Override
                     public Unit invoke(Boolean isLowMemory) {
+                        appDataCollector.setLowMemory(isLowMemory == Boolean.TRUE);
```

Comment:
Should this be checking the value of the instance instead of equality with the static constant? I highly doubt this would end up with [a constructed `Boolean()` instance](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#Boolean(boolean)) but still.

```suggestion
                        appDataCollector.setLowMemory(isLowMemory.booleanValue());
```